### PR TITLE
Correct overcloud password docs

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

**Comment by mpryc** [comment]
May I ask to rebase this change? I have not implemented yet rebase before CI run which we've discussed.

File: ansible/README.md [hunk 1]
```diff
@@ -178,10 +178,10 @@ You can also access the OSP console using your local web browser: <http://192.16
 | <!-- --> | <!-- --> |
 | ------------ | ------------ |
 | User | `admin` |
-| Pwd | The admin password can be found in the `/home/cloud-admin/tripleo-deploy/tripleo-overcloud-passwords.yaml` file on the `openstackclient` pod in the `openstack` namespace. |
+| Pwd | The admin password can be found in the `/home/cloud-admin/tripleo-deploy/clouds.yaml` file on the `openstackclient` pod in the `openstack` namespace. |
```
Comment (stuggi) [inline ansible/README.md:181]: with https://github.com/openstack-k8s-operators/osp-director-operator/pull/289/files this is now at the default location for user specific `clouds.yaml` at `/home/cloud-admin/.config/openstack/clouds.yaml`

File: ansible/README.md [hunk 2]
```diff
@@ -178,10 +178,10 @@ You can also access the OSP console using your local web browser: <http://192.16
...
 
 ```
-oc exec -it openstackclient -- cat /home/cloud-admin/tripleo-deploy/tripleo-overcloud-passwords.yaml | grep -w AdminPassword
+oc exec -it openstackclient -- cat /home/cloud-admin/tripleo-deploy/clouds.yaml | grep -w password
```
Comment (stuggi) [inline ansible/README.md:184]: `/home/cloud-admin/.config/openstack/clouds.yaml`

**Review by stuggi — CHANGES_REQUESTED** [review]

**Comment by dprince** [comment]
> May I ask to rebase this change? I have not implemented yet rebase before CI run which we've discussed.

Yes. I will rebase and fix it now.

File: ansible/README.md (hunk 1 above)
Comment (dprince) [reply ansible/README.md]: @stuggi got it, thanks!

**Review by stuggi — APPROVED** [review]
/lgtm

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**